src/stores/FileStore.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const alt = require('../alt');
+
+const FileActions = require('../actions/FileActions.js');
+
+class FileStore {
+  constructor() {
+
+    this.bindActions(FileActions);
+
+    this.state = {
+      fileName: ''
+    };
+
+  }
+
+  onClearName() {
+    this.setState({
+      fileName: ''
+    });
+  }
+
+  onUpdateName(evt) {
+    this.setState({
+      fileName: evt.target.value
```

Comment:
pull `value` out of the event before calling the action, I don't like events being passed through the dispatcher into the store, as it is a UI concern, not a data concern